src/schema/schema_worker.c

```diff
@@ -128,15 +128,15 @@ __schema_layered_worker_verify(WT_SESSION_IMPL *session, const char *uri,
           ingest_ret =
             __wt_schema_worker(session, ingest_uri, file_func, name_func, cfg, open_flags));
 
-        /*
-         * FIXME-WT-15433 Add an assertion that verifying the ingest table of the leader never
-         * returns EBUSY.
-         */
+        WT_ASSERT_ALWAYS(session, ingest_ret != EBUSY,
+          "Verify(layered): %s ingest table on leader cannot be verified. "
```

Comment:
np: If we mention the ingest table, we know it's layered.
```suggestion
          "Verify: %s ingest table on leader cannot be verified. "
```